CMakeLists.txt

```diff
@@ -68,11 +68,21 @@ endif()
 find_package(VulkanHeaders REQUIRED CONFIG)
 
 # Find and create the Vulkan::Vulkan library
-find_library(Vulkan_LIBRARY NAMES vulkan vulkan-1)
-add_library(Vulkan::Vulkan UNKNOWN IMPORTED)
-set_target_properties(Vulkan::Vulkan PROPERTIES
-    IMPORTED_LOCATION ${Vulkan_LIBRARY})
+add_library(Vulkan::Vulkan SHARED IMPORTED)
 target_link_libraries(Vulkan::Vulkan INTERFACE Vulkan::Headers)
+if (WIN32)
+    list(APPEND CMAKE_PROGRAM_PATH ${VULKAN_LOADER_INSTALL_DIR})
+
+    find_program(Vulkan_LIBRARY_DLL NAMES vulkan-1.dll)
```

Comment:
I dislike this. find_program is for finding executables. The following code should work.

```
find_library(Vulkan_LIBRARY_DLL NAMES vulkan-1.dll PATH_SUFFIXES bin)
```

Reply:
That does indeed work.